tests/unit_tests/test_lib/test_hashutil.py

```diff
@@ -1,99 +1,107 @@
+from __future__ import annotations
+
 import base64
 import hashlib
 from pathlib import Path
 
 import pytest
-from hypothesis import example, given
-from hypothesis import strategies as st
+from _pytest.fixtures import SubRequest
 from pyfakefs.fake_filesystem import FakeFilesystem
 from wandb.sdk.lib import hashutil
 
+BYTE_STRS: tuple[bytes] = (
+    b"",
+    b"\xb7\xb7",
+    b"\x89\xf3\xa2\xe1\xda",
+)
+
+TEXT_STRS: tuple[str] = (
+    "",
+    "foo",
+    "¬\U000f0c9a",
+)
+
+
+@pytest.fixture(params=[pytest.param(b, id=f"bin{i}") for i, b in enumerate(BYTE_STRS)])
+def bin_data(request: SubRequest) -> bytes:
+    """Binary data."""
+    return request.param
+
+
+@pytest.fixture(params=[pytest.param(s, id=f"txt{i}") for i, s in enumerate(TEXT_STRS)])
+def txt_data(request: SubRequest) -> str:
+    """Text data."""
+    return request.param
+
+
+@pytest.fixture(params=[pytest.param(b, id=f"bin{i}") for i, b in enumerate(BYTE_STRS)])
+def bin_data2(request: SubRequest) -> bytes:
+    """A second instance of binary data, if needed."""
+    return request.param
 
+
+# ------------------------------------------------------------------------------
 def test_md5_string():
     assert hashutil.md5_string("") == "1B2M2Y8AsgTpgAmY7PhCfg=="
     assert hashutil.md5_string("foo") == "rL0Y20zC+Fzt72VPzMSk2A=="
 
 
-@given(st.binary())
-def test_hex_to_b64_id(data):
-    hex_str = data.hex()
-    assert hashutil.hex_to_b64_id(hex_str) == base64.b64encode(data).decode("ascii")
+def test_hex_to_b64_id(bin_data):
+    hex_str = bin_data.hex()
+    assert hashutil.hex_to_b64_id(hex_str) == base64.b64encode(bin_data).decode("ascii")
 
 
-@given(st.binary())
-def test_hex_to_b64_id_bytes(data):
-    hex_bytes = data.hex().encode("ascii")
-    assert hashutil.hex_to_b64_id(hex_bytes) == base64.b64encode(data).decode("ascii")
+def test_hex_to_b64_id_bytes(bin_data):
+    hex_bytes = bin_data.hex().encode("ascii")
+    assert hashutil.hex_to_b64_id(hex_bytes) == base64.b64encode(bin_data).decode(
+        "ascii"
+    )
```

Comment:
nit: kind of an ugly line break. Creating a helper variable would be justified here IMO

```python
bin_data_b64 = base64.b64encode(bin_data).decode("ascii")
assert hashutil.hex_to_b64_id(hex_bytes) == bin_data_b64
```